spyder/plugins/appearance/widgets.py

```diff
@@ -8,7 +8,9 @@
 
 from qtpy.QtCore import Qt
 from qtpy.QtWidgets import (QDialog, QDialogButtonBox, QGridLayout, QGroupBox,
```

Comment:
```suggestion
from qtpy.QtWidgets import (
    QDialog,
    QDialogButtonBox,
    QGridLayout,
    QGroupBox,
    QHBoxLayout,
    QMessageBox,
    QVBoxLayout,
    QWidget,
)
```

- Use Black/Ruff style to avoid possible conflicts in the future.
- Also, organize imports alphabetically.